beacon_node/lighthouse_network/src/discovery/enr_ext.rs

```diff
@@ -232,6 +232,9 @@ impl CombinedKeyExt for CombinedKey {
                         .expect("libp2p key must be valid");
                 Ok(CombinedKey::from(ed_keypair))
             }
+            Keypair::Ecdsa(_) => {
+                todo!("What here??")
```

Comment:
@AgeManning I didn't remember until I did an extra review, but it seems this needs your input

Reply:
Looks like they've added a new keypair type. We haven't built the machinery to handle this, so just throw an error saying "invalid key type"